Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY automation/Extension/firefox/. ./
 COPY automation/Extension/webext-instrumentation/. ../webext-instrumentation
 RUN npm install
 RUN npm run build
+RUN cp dist/openwpm-1.0.zip openwpm.xpi
```

Comment:
I think we should use `COPY` here, instead of `RUN cp`?

Reply:
`COPY` moves files from the host system, while `RUN cp` copies filed around inside the image.
At least this is my understanding after reading the documentation

Reply:
Thus far we seem to have accidentally copied in the extension build on the host and never used the extension built inside of the image.

Reply:
Makes sense. Good work finding this!